go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/spf13/cobra v1.6.0
 	github.com/spruceid/siwe-go v0.2.1-0.20220804171946-fc1b0374f4ff
 	github.com/stretchr/testify v1.8.1
-	github.com/tablelandnetwork/sqlparser v0.0.0-20221209125022-863bbd9f6527
+	github.com/tablelandnetwork/sqlparser v0.0.0-20221229133649-a34f5eedb409
```

Comment:
This is the PR's latest commit. If every thing is ok, I'll merge that PR, then update this to the main latest commit.